packages/components/src/internal/components/administration/ProjectManagementPage.tsx

```diff
@@ -21,7 +23,14 @@ const Buttons: FC<RequiresModelAndActions> = memo(() => {
 });
 
 export const ProjectManagementPage: FC = memo(() => {
+    const [successMsg, setSuccessMsg] = useState<string>();
     const { user } = useServerContext();
+
+    useEffect(() => {
+        const successMessage = decodeURI(getLocation().query?.get('successMsg'));
```

Comment:
Maybe there's a better way to get the success msg to appear after redirect..?

Reply:
Rather than putting the entire message on the URL what if there is just a bit that this page receives on the URL and if it is set then it displays an associated banner message?

Reply:
I've changed the implementation—the success message is a bit specific, `<project name> successfully deleted.` but I'm now passing only the project name instead of the entire message

Reply:
Great, thanks. Could consider `Project "<project name>" successfully deleted.`.